examples/get_started/torch-loader.py

```diff
@@ -69,6 +69,7 @@ def forward(self, x):
         ds.to_pytorch(transform=transform),
         batch_size=25,
         num_workers=4,
+        persistent_workers=True,
```

Comment:
With `persistent_workers` set, it takes much less time to run 3 epochs than before.